src/components/payments/transfer/AmigoList.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import Amigo from "./AmigoTransfer";
+import { Container } from "@material-ui/core";
+
+export class AmigoList extends React.Component {
```

Comment:
Any reason for keeping class components here and using functional components in other files?